sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveWebHelper.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.spring.reactive;
+
+import io.sentry.IHub;
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import io.sentry.SystemOutLogger;
+import java.util.function.Consumer;
+import org.springframework.web.server.ServerWebExchange;
+
+public final class SentryReactiveWebHelper {
+  static final String REQUEST_HUB_ATTR_NAME = "SentryReactiveWebHelper.REQUEST_HUB_ATTR_NAME";
+
+  private static final ILogger LOGGER = new SystemOutLogger();
+
+  public static void withRequestHub(ServerWebExchange exchange, Consumer<IHub> hubConsumer) {
+    Object ihub = exchange.getAttributes().get(REQUEST_HUB_ATTR_NAME);
```

Comment:
can we call it just `hub` or the `i` has a different meaning here that I fail to see it?

Reply:
it's done 👍🏻